app/src/main/java/swati4star/createpdf/fragment/AddTextFragment.java

```diff
@@ -0,0 +1,303 @@
+        if (data == null || resultCode != RESULT_OK || data.getData() == null)
+            return;
+        if (requestCode == INTENT_REQUEST_PICK_PDF_FILE_CODE) {
+            mPdfUri = data.getData();
+            mPdfpath = RealPathUtil.getRealPath(getContext(), data.getData());
+            showSnackbar(mActivity, getResources().getString(R.string.snackbar_pdfselected));
+            return;
+        }
+        if (requestCode == INTENT_REQUEST_PICK_TEXT_FILE_CODE) {
+            mTextUri = data.getData();
+            mTextPath = RealPathUtil.getRealPath(getContext(), data.getData());
+            showSnackbar(mActivity, getResources().getString(R.string.snackbar_txtselected));
+        }
+        setTextAndActivateButtons(mPdfpath, mTextPath);
+    }
+
+    private void setTextAndActivateButtons(String pdfPath, String textPath) {
+        if (pdfPath == null || textPath == null) {
+            showSnackbar(mActivity, R.string.error_occurred);
+            resetView();
+            return;
+        }
+        mMorphButtonUtility.setTextAndActivateButtons(pdfPath,
+                mSelectPDF, mCreateTextPDF);
+        mMorphButtonUtility.setTextAndActivateButtons(textPath,
+                mSelectText, mCreateTextPDF);
+    }
+
+    public void resetView() {
+        mPdfpath = mTextPath = null;
+        mPdfUri = mTextUri = null;
+        mMorphButtonUtility.morphToGrey(mCreateTextPDF, mMorphButtonUtility.integer());
+        mCreateTextPDF.setEnabled(false);
+    }
+
+    private void addText(String fileName) {
+        String mStorePath = mSharedPreferences.getString(STORAGE_LOCATION,
+                getDefaultStorageLocation());
+        String mPath = mStorePath + fileName + pdfExtension;
+        try {
+            StringBuilder text = new StringBuilder();
+            BufferedReader br = new BufferedReader(new FileReader(mTextPath));
+            String line;
+
+            while ((line = br.readLine()) != null) {
+                text.append(line);
+                text.append('\n');
+            }
+            br.close();
+
+            OutputStream fos = new FileOutputStream(new File(mPath));
+
+            PdfReader pdfReader = new PdfReader(mPdfpath);
+            PdfStamper pdfStamper = new PdfStamper(pdfReader, fos);
+
+            PdfContentByte pdfContentByte = pdfStamper.getOverContent(pdfReader.getNumberOfPages());
+
+            // Add text in existing PDF
+            pdfContentByte.beginText();
+            pdfContentByte.setFontAndSize(BaseFont.createFont
+                            (BaseFont.TIMES_ROMAN, //Font name
+                                    BaseFont.CP1257, //Font encoding
+                                    BaseFont.EMBEDDED //Font embedded
+                            )
+                    , 12); // set font and size
+            pdfContentByte.setTextMatrix(10, 10); // set x and y co-ordinates
```

Comment:
for single line problem try removing this.
Refer `itext` documentation, see `PdfContentByte` class.
Try if can use `Paragraph` class. Refer `TextToPDF` creation for this.